src/SearchButton.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const SearchButton = () => {
+  return <button className="btn btn-primary">Search</button>;
```

Comment:
Do you think we could receive this `Search` via props and make this button more generic?

Reply:
Would it make more sense then! I am not quite sure?